kornia/augmentation/container/augment.py

```diff
@@ -168,12 +168,37 @@ class AugmentationSequential(TransformMatrixMinIn, ImageSequential):
         ...    random_apply=10,
         ... )
         >>> out = aug_list(input, mask, bbox)
+
+    How to use a dictionary as input with AugmentationSequential? The dictionary should starts with
+    one of the datakey availables.
+
+        >>> import kornia.augmentation as K
+        >>> img = torch.randn(1, 3, 256, 256)
+        >>> mask = [torch.ones(1, 3, 256, 256), torch.ones(1, 2, 256, 256)]
+        >>> bbox = [
+        ...    torch.tensor([[28.0, 53.0, 143.0, 164.0], [254.0, 158.0, 364.0, 290.0], [307.0, 204.0, 413.0, 350.0]]),
+        ...    torch.tensor([[254.0, 158.0, 364.0, 290.0], [307.0, 204.0, 413.0, 350.0]])
+        ... ]
+        >>> bbox = [Boxes.from_tensor(i).data for i in bbox]
+        >>> aug_dict = K.AugmentationSequential(
+        ...    K.ColorJiggle(0.1, 0.1, 0.1, 0.1, p=1.0),
+        ...    K.RandomHorizontalFlip(p=1.0),
+        ...    K.ImageSequential(K.RandomHorizontalFlip(p=1.0)),
+        ...    K.ImageSequential(K.ColorJiggle(0.1, 0.1, 0.1, 0.1, p=1.0)),
+        ...    data_keys=None,
+        ...    same_on_batch=False,
+        ...    random_apply=10,
+        ... )
+        >>> data = {'image': img, 'mask': mask[0], 'mask-b': mask[1], 'bbox': bbox[0], 'bbox-other':bbox[1]}
```

Comment:
can you update the docs / tutorials (later) stating how the keywords names work ? it handled properly to detect uniques keywords ?